.github/workflows/ci-workflow.yml

```diff
@@ -92,7 +92,13 @@ jobs:
           nohup bash -c "python /speculos/speculos.py bin/app.elf --sdk 2.0 --display headless" > speculos.log 2<&1 &
           cd tests && make install && make test
 
+      - name: Show speculos log
+        if: success() || failure()
+        run: |
+          cat speculos.log
+
       - name: Upload Speculos log
+        if: success() || failure()
```

Comment:
During test faillure we were not uploading the logs, this is just for better debugging.